_data/people/dynamic-entropy.yml

```
@@ -0,0 +1,11 @@
+active: true
+focus-area:
+  - doma
+  - osglhc
+institution: University of Wisconsin–Madison
+name: Rahul Chauhan
+photo: "/assets/images/team/Rahul-Chauhan.jpg"
+shortname: dynamic-entropy
+title: Research Cyberinfrastructure Specialist
+website:
```

```suggestion
website: https://github.com/dynamic-entropy
```

Just going to add your GitHub so there's some hyperlink associated with your name.